src/cio-config.ts

```diff
@@ -0,0 +1,32 @@
+export enum CioRegion {
+  US = 'US',
+  EU = 'EU',
+}
+
+export enum CioLogLevel {
+  None = 'none',
+  Error = 'error',
+  Info = 'info',
+  Debug = 'debug',
+}
```

Comment:
`CustomerioEnum` and `CustomerioConfig` have been merged into one file.
Another change you'll see here is the `CioLogLevel` now uses `None`, `Error`, `Info` and `Debug` instead of `none`, `error`, `info`, `debug`. This now matches with the other enums convention. It does not affect the functionality but the migrating users will have to make this change in their app to consume logLevls.